data/lib.js

```diff
@@ -17,6 +17,30 @@ export const areas = [
     country: "France",
     image: "/../public/images/onearm-hang-font.jpg",
   },
+  {
+    id: nanoid(),
+    name: "Avalonia",
+    routes: 485,
+    stone: "Sandstone",
+    country: "Germany",
+    image: "/../public/images/crimp-underhold.jpg",
+  },
+  {
+    id: nanoid(),
+    name: "Île-de-France",
+    routes: 30045,
+    stone: "Sandstone",
+    country: "France",
+    image: "/../public/images/onearm-hang-font.jpg",
+  },
+  {
+    id: nanoid(),
+    name: "Avalonia",
+    routes: 485,
+    stone: "Sandstone",
+    country: "Germany",
+    image: "/../public/images/crimp-underhold.jpg",
```

Comment:
Do you already use these images or are they important for later?

Reply:
I imported that one testwise but also I will use it later :)